spring-cloud-sleuth-brave/src/main/java/org/springframework/cloud/sleuth/brave/instrument/messaging/TraceConsumerPostProcessor.java

```diff
@@ -21,13 +21,17 @@
 
 import org.springframework.beans.factory.BeanFactory;
 import org.springframework.kafka.core.ConsumerPostProcessor;
+import org.springframework.util.ClassUtils;
 
 class TraceConsumerPostProcessor<K, V> implements ConsumerPostProcessor<K, V> {
 
 	private final BeanFactory beanFactory;
 
 	private KafkaTracing kafkaTracing;
 
+	// Because it's not public in Brave
```

Comment:
Should we open an issue for this in Brave?

Reply:
Good idea, although as far as I know that's fine by design